### PR TITLE
Fixes Bug 1196722 - allow FS to save raw, dumps & processed in combined call

### PR DESCRIPTION
**Pull request by twobraids**
removes out-dated prohibition on saving raw crash in the ```save_raw_and_processed``` method

### CODE REVIEW

File: socorro/unittest/external/fs/test_fsdatedradixtreestorage.py [hunk 1]
```diff
@@ -40,24 +40,159 @@ def _common_config_setup(self):
...
+                os.path.exists(
+                    "%s/%s.foo.dump" % (
+                        self.fsrts._get_radixed_parent_directory(self.CRASH_ID_1),
+                        self.CRASH_ID_1
+                    )
+                )
+            )
+        finally:
+            self.fsrts.remove(self.CRASH_ID_1)
+    def test_save_raw_crash_no_dumps(self):
```
Comment (AdrianGaudebert) [inline socorro/unittest/external/fs/test_fsdatedradixtreestorage.py]: missing newline here

Comment (AdrianGaudebert) [inline socorro/unittest/external/fs/test_fsdatedradixtreestorage.py]: Wait, this method has the exact same name as the one above?

**Comment by AdrianGaudebert** [comment]
Looking good to me. r+ when the tests pass.

**Comment by twobraids** [comment]
new line added, method name corrected, tests now passing

**Pull request merged**